lib/internal/per_context/domexception.js

```diff
@@ -49,13 +53,22 @@ function ensureInitialized() {
   forEachCode((name, codeName, value) => {
     nameToCodeMap.set(name, value);
   });
+
+  // These were removed from the error names table.
+  // See https://github.com/heycam/webidl/pull/946.
+  disusedNamesSet = new SafeSet([
+    'DOMStringSizeError',
+    'NoDataAllowedError',
+    'ValidationError',
+  ]);
```

Comment:
Passing an array to `SafeSet` is not safe as it relies on the globally mutable `Array.prototype[Symbol.iterator]`
```suggestion
  disusedNamesSet = new SafeSet()
    .add('DOMStringSizeError')
    .add('NoDataAllowedError')
    .add('ValidationError');
```

Does it need to be a `Set`? Can't we do this with an array or even a simply a "IF…OR…OR…OR" pattern?

Reply:
These APIs are actually one of the cases this sort of guarding is important for spec compliance.